js/main.js

```diff
@@ -1,2 +1,162 @@
 $(function () {
-})
+
+	let botChoice
+	let userChoice
+	let botScore = 0
+	let userScore = 0
+	let botWins = 0
+	let userWins = 0
+	resetGame()
+
+	$('button').click((event) => {
+
+		//Calling all functions
+		clearBonusMessage()
+		assignBotChoice()
+		assignUserChoice()
+		determineWinner()
+		bonusPoints()
+		roundOneWinner()
+		roundTwoWinner()
+	})
+
+	//Randomly assigning a choice to the bot
+	function assignBotChoice() {
+		//Random number from 1 to 3
+		botChoice = Math.floor(Math.random() * 3) + 1
+
+		//If statement to assign numbers to string choices
+		if (botChoice === 1) {
+			botChoice = 'rock'
+		} else if (botChoice === 2) {
+			botChoice = 'scissors'
+		} else if (botChoice === 3) {
+			botChoice = 'paper'
+		} else {
+			console.log('Invalid choice.')
+		}
```

Comment:
@cswebdesign `botChoice` will either be 1, 2, or 3. if it's not 1 or 2, then we can assume it'll be 3, so `line 33` could be replaced with a simple else statement. `line 35` will never run, but it's great that you're thinking about edge cases and error handling. 👍 

An alternative solution could be to store the choices in an array and access them via index (minus the 1). 
check solution [here](https://github.com/jsr20190826/rock-paper-scissors-solution/blob/master/js/main.js#L4)